### ISSUE TITLE
Failed: FtpMoveSink should move a file

### ISSUE THREAD

**Issue by ennru** [akka/alpakka#1492]
```
[info] FtpMoveSink
[info] - should move a file *** FAILED *** (606 milliseconds)
[info]   IOResult(1,Failure(javax.net.ssl.SSLException: bad record MAC)) was not equal to IOResult(1,Success(Done)) (CommonFtpStageSpec.scala:310)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:346)
[info]   at org.scalatest.Matchers$AnyShouldWrapper.shouldBe(Matchers.scala:6864)
[info]   at akka.stream.alpakka.ftp.CommonFtpStageSpec$$anonfun$7$$anonfun$apply$mcV$sp$20.apply(CommonFtpStageSpec.scala:310)
[info]   at akka.stream.alpakka.ftp.CommonFtpStageSpec$$anonfun$7$$anonfun$apply$mcV$sp$20.apply(CommonFtpStageSpec.scala:301)
```

https://travis-ci.org/akka/alpakka/jobs/491569258#L957

**Comment by tgambet** [comment]
Should be closed since #1668.

**Comment by ennru** [comment]
Thank you.
Expected to be solved with #1668.
